tests/test_integrated_channels/test_integrated_channel/test_transmitters/test_learner_data.py

```diff
@@ -108,3 +108,33 @@ def test_raises_client_error_on_status_code(self, is_already_tx, mock_lms_id):
             remote_user_id='user_id'
         )
         self.learner_transmitter.handle_transmission_error.assert_called_once()
+
+    def test_learner_data_transmission_feature_flag(self):
+        """
+        Test that a customer's configuration can disable learner data transmissions
+        """
+        # Set feature flag to true
+        self.enterprise_config.disable_learner_data_transmissions = True
+
+        self.learner_transmitter.client.create_assessment_reporting = Mock()
+        self.learner_transmitter.client.create_course_completion = Mock()
+
+        LearnerExporterMock = LearnerExporter
+        self.learner_transmitter.single_learner_assessment_grade_transmit(
+            LearnerExporterMock,
+            remote_user_id='user_id'
+        )
+        assert not self.learner_transmitter.client.create_assessment_reporting.called
+
+        self.learner_transmitter.assessment_level_transmit(
```

Comment:
I could theoretically call both assessment level transmission jobs and then assert that `create_assessment_reporting` is never called but I'm asserting twice for readability